packages/ui/src/helpers/storage/file-uploader/index.ts

```diff
@@ -0,0 +1 @@
+export * from './utils';
```

Comment:
Can we stick to `camelCase` for file naming?

Reply:
The utils folder matches the same pattern we have in the authenticator with its utils.ts file. Is the problem the `file-uploader` folder? Or do you prefer I rename the uploader utils file?

Reply:
The folder name, good catch

Reply:
I removed `File-Uploader` for `fileUploader` in the React components